Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -1128,6 +1152,53 @@ private int broadcastMessage(final IUser sender, final String permission, final
         return players.size();
     }
 
+    @Override
+    public void broadcastTl(final String tlKey, final Object... args) {
+        broadcastTl(null, null, true, tlKey, args);
+    }
+
+    @Override
+    public void broadcastTl(final IUser sender, final String tlKey, final Object... args) {
+        broadcastTl(sender, null, false, tlKey, args);
+    }
+
+    @Override
+    public void broadcastTl(final IUser sender, final String permission, final String tlKey, final Object... args) {
+        broadcastTl(sender, u -> !u.isAuthorized(permission), false, tlKey, args);
+    }
+
+    @Override
+    public void broadcastTl(IUser sender, Predicate<IUser> shouldExclude, String tlKey, Object... args) {
+        broadcastTl(sender, shouldExclude, false, tlKey, args);
+    }
+
+    @Override
+    public void broadcastTl(final IUser sender, final Predicate<IUser> shouldExclude, final boolean parseKeywords, final String tlKey, final Object... args) {
+        if (sender != null && sender.isHidden()) {
+            return;
+        }
+
+        for (final User user : getOnlineUsers()) {
+            if (sender != null && user.isIgnoredPlayer(sender)) {
+                continue;
+            }
+
+            if (shouldExclude != null && shouldExclude.test(user)) {
+                continue;
+            }
+
+            final Object[] processedArgs;
+            // FUCK THIS STUPID BULLSHIT
```

Comment:
```suggestion
```

Reply:
i find this comment to be useful for future maintainers